mixer/adapter/stdio/config/config.proto

```diff
@@ -118,4 +119,23 @@ message Params {
     // The maximum number of old rotated log files to retain.  The default
     // is to retain at most 1000 logs. 0 indicates no limit.
     int32 max_rotated_files = 9;
+
+    // Controls the level of sampling that will be used by a stdio handler for logs entries.
+    // Sampling is configured based on a duration, with the following logic:
+    //  - over the course of the duration, `max_unsampled_entries` are allowed unmodified
+    //  - after `max_unsampled_entries`, for the remainder of the duration a single log entry will
+    //    be emitted for every `sampling_rate` entries received.
+    message LogsSampling {
```

Comment:
I'm not liking the S in there. I think LogSampling would be better.

Reply:
renamed everything to remove the 's'(es).